### ISSUE TITLE
[Question] Does urlint account for broken anchor links (website.com/link#anchor)?

### ISSUE THREAD

**Issue by pruett** [urlint/urlint#122]
Quickly glancing at the source code it wasn't clear whether or not this library accounts for checking whether anchor links are broken or not. In other words, does `urlint` check to see if the link's resolved content has an associated `id` element with a value of `anchor` in its markup, for the hypothetical link of `link.com/page#achor`?

**Comment by Kikobeats** [comment]
not really since the anchor is a front end UX behavior

```
headers https://example.com\#test
HTTP/2 200
age: 328719
cache-control: max-age=604800
content-type: text/html; charset=UTF-8
date: Sat, 04 Apr 2020 11:09:37 GMT
etag: "3147526947+ident"
expires: Sat, 11 Apr 2020 11:09:37 GMT
last-modified: Thu, 17 Oct 2019 07:18:26 GMT
server: ECS (mic/9ABC)
vary: Accept-Encoding
x-cache: HIT
content-length: 1256
```

note how example.com is reachable but that doesn't matter `#test` is a thing 🙂

**Comment by pruett** [comment]
Right. It would require parsing the successful request’s contents to see if an `id` or `name` attribute matches the provided anchor. I understand the status code of 200 since it is a successful request, but it would be great to do the extra introspection for any links that do include an anchor.

**Comment by Kikobeats** [comment]
good idea! I reopened the issue.

Since the process get the HTML, it should be trivial try to find the `id` selector into the HTML.

Maybe I can write a standalone library and reuse it here

**Comment by pruett** [comment]
If this is a feature of interest, I'd be interested in attempting to add it. Just let me know.

**Comment by pruett** [comment]
Missed your comment above. Even better!!
